### PR TITLE
feat: create module

### PR DESCRIPTION
**Pull request by ztzxt**

### CODE REVIEW

File: glue.tf [hunk 1]
```diff
@@ -0,0 +1,203 @@
+resource "aws_glue_catalog_table" "this" {
+  provider      = aws.glue
+  count         = var.access_logs_enabled ? 1 : 0
+  name          = "${module.this.namespace}_${replace((var.name), "-", "_")}"
+  database_name = "${module.this.environment}_alb_logs"
+  table_type    = "EXTERNAL_TABLE"
+
+  partition_keys {
+    name = "day"
```
Comment (applike-ss) [inline glue.tf:9]: can we rename this to date instead?

File: providers.tf [hunk 2]
```diff
@@ -0,0 +1,3 @@
+provider "aws" {
+  alias = "glue"
+}
```
Comment (applike-ss) [inline providers.tf:3]: is this definition still needed even though you configured this alias already in versions.tf?

File: variables.tf [hunk 3]
```diff
@@ -0,0 +1,112 @@
+variable "access_logs" {
+  description = "Map containing access logging configuration for load balancer."
+  type        = map(string)
```
Comment (applike-ss) [inline variables.tf:3]: can we specify this more strictly?

File: variables.tf [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+  description = "A boolean flag to enable/disable access_logs"
+}
+
+variable "access_logs_prefix" {
+  type        = string
+  default     = ""
+  description = "The S3 log bucket prefix"
+}
+
+variable "aws_account_id_monitoring" {
```
Comment (applike-ss) [inline variables.tf:25]: can we make this `aws_account_id_logging`?

File: glue.tf (hunk 1 above)
Comment (ztzxt) [reply glue.tf:9]: i'd like to stick with day here to keep it consistent with AWS docs

**Review by applike-ss — APPROVED** [review]

**Pull request merged**